r/tests/testthat/test-dplyr-group-by.R

```diff
@@ -166,3 +166,67 @@ test_that("group_by() with namespaced functions", {
     tbl
   )
 })
+
+test_that("group_by() with .add", {
```

Comment:
My intention was not to make `.add = FALSE` explicit so that detect if the dplyr's default behavior changed, but it is unlikely that this is the case, so it seems fine to make it explicit.